sermant-plugins/sermant-service-registry/spring-cloud-registry-plugin/src/main/java/com/huawei/registry/declarers/cloud3/x/ZookeeperInstanceSupplierDeclarer.java

```diff
@@ -24,23 +24,24 @@
 import com.huaweicloud.sermant.core.plugin.agent.matcher.MethodMatcher;
 
 /**
- * 针对springCloud 3.x获取实例列表兼容
+ * Compatible with obtaining instance lists for springCloud 3.x
  * <p>
- * 自从3.x版本，zookeeper注册发现新增InstanceSupplier进行实例获取， 其将包裹原来的DiscoveryClient，因此原来的拦截点将不再受用
+ * Since version 3.x, zookeeper registers and discovers the new InstanceSupplier for instance acquisition, which will
```

Comment:
InstanceSupplier is added to the ZooKeeper registration and discovery to obtain instances.

Reply:
done